src/test/config.sol

```diff
@@ -95,8 +95,8 @@ contract Config {
         // Values for spell-specific parameters
         //
         spellValues = SpellValues({
-            deployed_spell:                 address(0x15c263fda6Cb175878d523aCc095be6f1bCe1509),     // populate with deployed spell if deployed
-            deployed_spell_created:         1670448300,              // use `./scripts/get-created-timestamp.sh <deployment-tx>`
+            deployed_spell:                 address(0xa8C4C020A86EA07b022a9a76E1959bbd12F126D5),     // populate with deployed spell if deployed
+            deployed_spell_created:         1671019944,              // use `./scripts/get-created-timestamp.sh <deployment-tx>`
```

Comment:
is this a blank spell you're using for testing? if so, maybe we go to `address(0)` here?

Reply:
This is the same blank spell there is in the file but deployed.

Reply:
ok, and since we're not casting it, we shouldn't archive? (just don't want to have to reverse engineer this when we've forgotten the context)

Reply:
I was actually thinking on casting it to test the script that I did modifications but archiving it seems unnecessary honestly. But if we prefer to not cast it is fine as well.

Reply:
yeah i'm probably making something out of nothing. you're right, there is value in the casting and lets just ignore the archive